### PR TITLE
#96 Profile Mobile View

### LINKED ISSUE #96: [Profile] [FE] Mobile view
## Context

The profile page allows users to view and edit their fostering preferences. Currently, the page functions as intended on desktop, but the mobile view is broken.

## Requirements

Reference the mobile design on the Figma and implement the mobile view. Try to reuse the same components, just changing the [responsive styles](https://chakra-ui.com/docs/styled-system/responsive-styles). This minimizes any issues we may have down the line if we update the page later.

#### PM

- [ ] Product-specific requirements
- [ ] Ensure the mobile view has the same or equivalent user experience that the desktop has
- [ ] Allow editing then canceling or saving changes successfully for the mobile view
- [ ] Name and email shouldn't be editable
- [ ] The profile should be visible in mobile view (it currently isn't)

#### EM

- [ ] Profile page on mobile is functional and works the same as desktop

#### Designer

- [ ] Design-specific requirements

### PR DESCRIPTION
**Pull request by laurenmcp**
##Profile Mobile View

Issue Number(s): #96.

What does this PR change and why?
Update responsive styling of profile components so that the profile appears as expected on both web and mobile screens.
### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria are met
- [ ] Database schema docs have been updated or are not necessary
- [ ] Code follows design and style guidelines
- [ ] Commits follow guidelines (concise, squashed, etc)
- [x] Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes

None

### Related PRs

None 

### Testing

Enumerate steps to test the functionality of your ticket. This should include edge cases and testing of error handling, if applicable.

- Profile page matches Figma designs on mobile while preserving existing appearance for web.
- Profile page functionality is unaffected by styling changes.

### CODE REVIEW

File: web/pages/profile.tsx [hunk 1]
```diff
@@ -525,6 +513,48 @@ function Profile() {
...
+          variant="solid-primary"
+          width={["50%", "100%"]}
+          onClick={async () => {
+            const req = await updatePreferences.mutateAsync({
+              uid: userData!.uid,
+              updateFields: preferences,
+            });
+            if (req.success) {
+              setEditing(false);
+            }
```
Comment (katsukixyz) [inline web/pages/profile.tsx]: Optional: you could add logic to display a toast if the request isn't successful. This is out of scope for your ticket, so no need to worry about it!

**Review by katsukixyz — APPROVED** [review]
Awesome work, everything looks good on the engineering side! Look out for comments from the rest of the LT.

**Comment by winniezhangg** [comment]
This looks great Lauren! Designs are consistent on desktop and mobile. Great work!!
